compiler/scenario-service/server/src/main/scala/com/digitalasset/daml/lf/scenario/Context.scala

```diff
@@ -149,6 +150,16 @@ class Context(val contextId: Context.ContextId) {
   def allPackages: Map[PackageId, Ast.Package] =
     extPackages + (homePackageId -> Ast.Package(modules, extPackages.keySet, None))
 
+  private val nextTransactionSeedAndSubmissionTime = {
+    // we seed crypto.Hash.secureRandom with time to get different coid at each run.
+    // This is fine
```

Comment:
This comment doesn’t explain why it’s fine. Presumably because you remap to node ids which are not going to differ between each run.

Reply:
I change to fix hash and time.